deployment/clouddeploy/gke-workers/base/gen-cperepos-map.yaml

```diff
@@ -0,0 +1,33 @@
+apiVersion: batch/v1
+kind: CronJob
+metadata:
+  name: gen-cperepos-map
+spec:
+  schedule: "0 6 */1 * *"
+  concurrencyPolicy: Forbid
+  jobTemplate:
+    spec:
+      activeDeadlineSeconds: 86400
+      template:
+        spec:
+          containers:
+          - name: gen-cperepos-map
+            image: gen-cperepos-map
```

Comment:
This image needs to match `build-and-stage.yaml`, where it's called `cpe-repo-gen`
https://github.com/google/osv.dev/blob/cd244c5cd936674ce7dac0baec6eeb80f65394f8/deployment/build-and-stage.yaml#L187

Reply:
Thank you for catching that. Fixed in [69122a9](https://github.com/google/osv.dev/pull/1171/commits/69122a9771b00b4fc6ef73f3d195214859c1d304)